ewah/ewah_utils/yml_loader.py

```diff
@@ -15,12 +18,15 @@
 Accessed 2020-07-28
 """
 
+with create_session() as session:
+    af_val = {var.val for var in session.query(Variable)}
 
 class Loader(base_loader):
     def __init__(self, stream):
+        currencies = af_val
         self._root = os.path.split(stream.name)[0]
         # Enable Jinja2 in the yaml files
-        yaml_stream = StringIO(Template(stream.read()).render(env=os.environ))
+        yaml_stream = StringIO(Template(stream.read()).render(env=os.environ, currencies=currencies))
```

Comment:
Logic (blocking): Rather than loading specificaly `currencies`, we should abstract this such that it will load all airflow variables as `airflow_variables` dictionary; `currencies` will then just be one variable that can be accessed in the `.yml` file.